pytorch_lightning/profiler/pytorch.py

```diff
@@ -426,12 +426,13 @@ def stop(self, action_name: str) -> None:
 
             def on_trace_ready(profiler):
                 if self.dirpath is not None:
+                    filename = self._prepare_filename(extension="")
                     if self._export_to_chrome:
-                        handler = tensorboard_trace_handler(self.dirpath, self._prepare_filename(extension=""))
+                        handler = tensorboard_trace_handler(self.dirpath, filename + "-" + str(action_name))
```

Comment:
Why not add the `action_name` keyword argument to `prepare_filename`?

Reply:
Done !